microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+
+1. Click the **Address bar and search** section.  The **Address bar and search** settings page<!-- edge://settings/search --> opens.
+
+1. Click the **Search suggestions and filters** section.  The **Search suggestions and filters** settings page<!-- edge://settings/searchFilters --> opens.
+
+1. Use the toggle **Show me search and site suggestions using my typed characters**:
+
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/search-suggestions-typed-chars.png)
+
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Microsoft Search in Bing
+
+If you're signed into Microsoft Edge using your work or school account (Microsoft Entra ID account), your typed-in characters are also sent to Microsoft Search in Bing, which returns work- or school-related results and cross-device suggested queries to you.
+
+Your administrator can choose to turn on or off the work- or school-related results or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` Microsoft Search in Bing (MSB) policy.  If your administrator turns off the `AddressBarMicrosoftSearchInBingProviderEnabled` policy:
+* In Microsoft Edge on your present device:
+   * You don't see the work- or school- related results from Microsoft Search in Bing.
+   * You see the query suggestions that are returned from your default search engine.
+* In Microsoft Edge on your other devices:
+   * You don't see the work- or school-related results.
+   * You don't see the query suggestions that you previously searched.
+
+You can further protect your privacy by doing any of the following:
+
+* Turn off the suggested queries entirely, by using the toggle **Show me search and site suggestions using my typed characters**, as described in [Controlling whether typed characters are sent](#controlling-whether-typed-characters-are-sent), above.
+
+* Keep the suggested-queries function, but don't send information about the suggested queries to Microsoft.  To prevent sending this information to Microsoft, go to `edge://settings/privacy`, and then under **Search and service improvement**, turn off the toggle **Help improve Microsoft products by sending the results from searches on the web**.  This data is never associated with you or your device, even if you turn on this setting.
+
+* Delete any query suggestion that you don't like, directly in the Address bar, by using the **X** button on the right side when hovering your mouse over the suggestion.
```

Comment:
I'm not sure this belongs in this list (but open to discussion). It sounds like it's more of helping you improve the validity of suggestions than improving the privacy of users.

Reply:
compare new paragraph & screenshot at bottom of 
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3224#deleting-data
![image](https://github.com/user-attachments/assets/70d5eea8-b2b5-4d48-a4d9-0f688d71d3c1)
"To delete history data from both Microsoft Edge and the server, for an individual suggestion result, in the list in the Address bar, click X to the right of a Recent searches history item:"

Reply:
added hidden todo comment:
```
* Delete any query suggestion that you don't like, directly 
in the Address bar, by using the **X** button on the right 
side when hovering your mouse over the suggestion.<!-- todo: remove 
list item?  might not belong in this list, b/c this item sounds more 
about helping you improve validity of suggestions than improving 
privacy.  also, it's covered in "Deleting data" section below, with 
screenshot: "To delete history data from both Microsoft Edge and the 
server, for an individual suggestion result, in the list in the Address bar, 
click **X** to the right of a **Recent searches** history item:" (.png).  
if delete this list item, reword list lead-in -->
```

Reply:
removed list item and rewrote list as paragraph:
```
To further protect your privacy, you can keep 
the suggested-queries function, without sending 
information about the suggested queries to 
Microsoft.  This data is never associated with you 
or your device, even if you turn on this setting.  To 
prevent sending information about the suggested 
queries to Microsoft, select **Settings and more** 
(![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy, search, and services**, 
and then in the **Search and service improvement** 
section, turn off the toggle **Help improve Microsoft 
products by sending the results from searches on the web**.
```